tests/flows/steps/test_setup_context.py

```
@@ -90,14 +90,43 @@ def test_setup_context_invalid_aws_credentials(
     context = PurchaseContext.from_order_data(order)
-    step = SetupContext(config, SWO_EXTENSION_MANAGEMENT_ROLE)
+    step = SetupContext(config)
 
     step(mpt_client_mock, context, next_step_mock)  # act
 
     next_step_mock.assert_not_called()
     assert "failing to provide valid AWS credentials" in one_time_notification_mock.call_args[0][1]
 
 
+def test_setup_context_invalid_apn_credentials(
+    mocker, config, aws_client_factory, order_factory, fulfillment_parameters_factory
```

Ruff (0.14.13)

Unused function argument: `aws_client_factory`

(ARG001)